### PR TITLE
Add Support For Password and Chip Input

### PR DESCRIPTION
**Pull request by lukeburpee**

### CODE REVIEW

File: src/ChipInput.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import ChipInput from 'material-ui-chip-input';
```
Comment (mihirsoni) [inline src/ChipInput.js]: Could you please add unit test cases for both the component ?

File: src/PasswordField.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Password from 'material-ui-password-field';
```
Comment (mihirsoni) [inline src/PasswordField.js]: Could you add unit test cases.

**Comment by mihirsoni** [comment]
@lukeburpee Have you closed this by mistake ?

**Comment by lukeburpee** [comment]
I actually opened this by mistake. I was trying to commit to my local fork,  but don't have much experience with github desktop. Sorry about that.

On another note, the werterbyte password field relies on the window for one of its utilities, so it doesn't go well with SSR. I've re-wrote it based on the material-ui example, but it relies on local state to change the password field visibility (not really the redux way, but probably best for working with redux-form). Still, it works and the unit tests should be basically the same as the TextField tests. I can create a pull request for that one separately if you like, but the chip input still needs work. I've seen a couple of different ways it could be done, and I haven't decided which would be the best approach. The wertarbyte chip input uses the "addChip" and "deleteChip" properties for controlled components, which throws off managing the input through "onChange". Shouldn't be too difficult, just have to find time to do it.

**Pull request closed without merging**